Automattic/wp-calypso
# Framework: update lib/automated-transfer interface and remove usage (try two)

Reverts Automattic/wp-calypso#13199

Alternative to #13139 This PR removes SitesList usage from `lib/automated-transfer` by updating the library interface.

### Testing Instructions
- /themes and /plugins loads correctly
- Prep a site that has a custom domain + business plan with you as the only user
- Set the custom domain as primary
- Wait for the SSL cert
- Existing users shouldn't see any options to transfer in /plugins or /themes
- Set abtest to `enabled` for `automated-transfer2` if you match the conditions above you should be able to transfer
